BencodeNET/Torrents/Torrent.cs

```diff
@@ -227,8 +227,8 @@ public virtual BDictionary ToBDictionary()
 
             var trackerCount = Trackers.Flatten().Count();
 
-            if (trackerCount > 0)
-                torrent[TorrentFields.Announce] = new BList(Trackers.First().Select(x => new BString(x, Encoding)));
+            if (trackerCount == 1)
```

Comment:
I think it's best to always add the `announce` field for compatability (`announce-list` is an optional extension) so we should keep `trackerCount > 0`.